scripts/mcp_detector_correction.py

```diff
@@ -40,6 +40,14 @@
     output_dir = args["<output_dir>"]
     skip_first_last_img = args["--skipimg"]
     verbose = args["--verbose"]
+
+    # in some rare instances, the MCP creates a duplicate set of the run in the same folder
+    # we need to only consider the first set in the autoreduction
```

Comment:
If this is something that occurs very often, we should have a pre-processing to handle the duplicates.
As a quick fix, these changes are okay.
I would recommend writing a new defect/story to address this properly.